packages/vscode-ui5-language-assistant/src/configuration.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  ConfigurationChangeEvent,
+  Event,
+  WorkspaceConfiguration,
+} from "vscode";
+import { CHANGE_LOG_LEVEL_REQUEST } from "@ui5-language-assistant/language-server";
+
+export const LOGGING_LEVEL_CONFIG_PROP = "UI5LanguageAssistant.logging.level";
+
+export function listenToLogLevelChanges<R>(opts: {
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any -- signature with `any` originates from VSCode APIs */
+  subscriptions: { dispose(): any }[];
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any -- signature with `any` originates from VSCode APIs */
```

Comment:
It might originate from vscode but you can still use `unknown` if you only send the parameter to the function

Reply:
thanks